### ISSUE TITLE
Weights shape for CRNN for TF are incorrect

### ISSUE THREAD

**Issue by meetshah1995** [keunwoochoi/music-auto_tagging-keras#4]
The CRNN takes a `96*1366` spectrogram image as input.  However the weights (for input batch normalization layer) provided for Tensorflow for CRNN have the shape of the 

```
bn_0_freq_beta_1 (1440,)
bn_0_freq_gamma_1 (1440,)
bn_0_freq_running_mean_1 (1440,)
bn_0_freq_running_std_1 (1440,)
```

which is incompatible with the input image size.

**Comment by keunwoochoi** [comment]
It is compatible because there is [zero padding](https://github.com/keunwoochoi/music-auto_tagging-keras/blob/master/music_tagger_crnn.py#L86).
